docs/11-circleci/01-getting-started.mdx

```diff
@@ -0,0 +1,150 @@
+---
+toc_max_heading_level: 4
+---
+
+# Getting started
+
+This guide will help you test and build your Unity project using the
+[Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity).
+
+## Overall steps
+
+1. Understand how CircleCI works.
+2. Set up a workflow for your project.
+3. Download the artifacts from your first build using the orb.
+
+## CircleCI
+
+If you are not familiar with CircleCI, read the official documentation on how to set up a
+[workflow](https://circleci.com/docs/getting-started). Subsequent steps will assume you have done
+so.
+
+#### Unity Orb
+
+[Orbs](https://circleci.com/orbs/) are reusable snippets of code that help automate repeated
+processes, speed up project setup and make it easy to integrate with third-party tools.
+
+To streamline the process of testing and building Unity projects in CircleCI, GameCI provides the
+[Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity) — a certified orb capable of
+dependencies caching, artifacts storage, test data collection and macOS, Windows, Docker, Machine
+and Runner support.
+
+In further sections, you will learn how to import and use the orb to test and build your project for
+several target platforms.
+
+_**Note:** the orb is in the early stages of development; it doesn't have all features included in
+the [Builder Action](/docs/github/builder)._
```

Comment:
Unless I'm missing something, the Orb and the GitLab solution should be equal in terms of features. And I have reservations about saying that the Orb is a WIP in the docs because, technically, it will always be until we drop support.

I feel like "early stages of development" gives the notion of ready to be used but not as complete as the alternative, which is the GH action in this case. All that said, maybe we could reword it like this:

```mdx
_**Note:** the Orb is in the early stages of development. If you need a feature-complete solution, check our GitHub
[Builder Action](/docs/github/builder)._
```

Let me know what you think!

Reply:
Ah, I must have misinterpreted what you typed. Early stages of development sounds like there are still bugs or that you don't have feature parity yet. Generally all our projects are on a best-effort basis and still in active development.

Your suggestion looks good to me though.

Reply:
> Early stages of development sounds like there are still bugs or that you don't have feature parity yet

That's fair. I'll remove the GitLab bit and push your suggestion.